docs/faq.rst

```diff
@@ -124,10 +124,28 @@ Which ML libraries does Fairlearn support?
     `predict()` methods. Also, any classification or regression
     algorithm can be evaluated using our metrics.
 
+Does Fairlearn support multi-class classification?
+    Certain metrics will still work for multi-class classification, e.g.,
+    accuracy score. For others it is more complicated since they rely on
+    binary classes, e.g., true positive rate.
+    There are
+    `ongoing discussions within the community <https://github.com/fairlearn/fairlearn/issues/752>`_
+    to add more extensive support to Fairlearn's assessment capabilities.
+    If you have specific thoughts feel free to add them to the discussion.
```

Comment:
Inviting 'specific thoughts' might make people concerned that their thoughts are insufficiently precise....